package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "nictiz.fhir.nl.stu3.eafspraak",
-    "version": "1.0.2",
+    "version": "1.0.3",
     "canonical": null,
     "web": "https://informatiestandaarden.nictiz.nl/wiki/MedMij:V2020.1/OntwerpeAfspraak",
```

Comment:
I only see an upped version number, the faulty key is not updated. I'll correct that.